modules/permutiveRtdProvider.js

```diff
@@ -174,6 +159,21 @@ function updateOrtbConfig (currConfig, segmentIDs, sspSegmentIDs, transformation
   const updatedUserKeywords = (currentUserKeywords === '') ? keywords : `${currentUserKeywords},${keywords}`
   deepSetValue(ortbConfig, 'ortb2.user.keywords', updatedUserKeywords)
 
+  // Set bidder specific extensions
+  if (bidder === 'rubicon') {
+    if (segmentIDs.length > 0) {
+      deepSetValue(ortbConfig, 'ortb2.user.ext.data.' + PERMUTIVE_STANDARD_KEYWORD, segmentIDs)
```

Comment:
this may be a dumb question - why is this set only for rubicon? with my naivete I'd expect it to be standardized and useful for everyone, or superfluous and the responsibility of rubicon to deduce from `user.data`.

Reply:
It's specific for Rubicon because this is how they read from the ortb2 object and it falls under `ortb2.user.ext.data`, where the `.ext` being they key part of the path. Following the OpenRTB 2.5 specification, we must place bidder-specific extensions in `.ext` properties.

I hope that makes sense 😬

Reply:
It's also worth pointing out, this data already exists under `ortb2.user.data`. We're just adding it into additional places in the ortb2 object for a bidder.

Reply:
The goal is, the word 'rubicon' should not appear in your code.